exporter/lokiexporter/README.md

```diff
@@ -6,13 +6,14 @@
 | Stability     | [deprecated]: logs   |
```

Comment:
Do you want to make it as not deprecated?

Reply:
I believe it still should be deprecated, we intend to have as many users migrate to otlp/otlphttp exporters as possible, however there are still some issues with loki's otlp ingestion that prevent some users from making that migration yet.